subsys/bluetooth/mesh/brg_cfg_cli.c

```diff
@@ -31,47 +191,170 @@ const struct bt_mesh_model_cb _bt_mesh_brg_cfg_cli_cb = {
 int bt_mesh_brg_cfg_cli_subnet_bridge_get(uint16_t net_idx, uint16_t addr,
 					  enum bt_mesh_subnet_bridge_state *status)
 {
-	return 0;
+	BT_MESH_MODEL_BUF_DEFINE(msg, OP_SUBNET_BRIDGE_GET, 0);
+	struct bt_mesh_msg_ctx ctx = BT_MESH_MSG_CTX_INIT_DEV(net_idx, addr);
+	const struct bt_mesh_msg_rsp_ctx rsp_ctx = {
+		.ack = &cli->ack_ctx,
+		.op = OP_SUBNET_BRIDGE_STATUS,
+		.user_data = status,
+		.timeout = msg_timeout,
+	};
+
+	bt_mesh_model_msg_init(&msg, OP_SUBNET_BRIDGE_GET);
+
+	return bt_mesh_msg_ackd_send(cli->model, &ctx, &msg, !status ? NULL : &rsp_ctx);
 }
 
 int bt_mesh_brg_cfg_cli_subnet_bridge_set(uint16_t net_idx, uint16_t addr,
 					  enum bt_mesh_subnet_bridge_state val,
 					  enum bt_mesh_subnet_bridge_state *status)
 {
-	return 0;
+	BT_MESH_MODEL_BUF_DEFINE(msg, OP_SUBNET_BRIDGE_SET, 1);
+	struct bt_mesh_msg_ctx ctx = BT_MESH_MSG_CTX_INIT_DEV(net_idx, addr);
+	const struct bt_mesh_msg_rsp_ctx rsp_ctx = {
+		.ack = &cli->ack_ctx,
+		.op = OP_SUBNET_BRIDGE_STATUS,
+		.user_data = status,
+		.timeout = msg_timeout,
+	};
+
+	bt_mesh_model_msg_init(&msg, OP_SUBNET_BRIDGE_SET);
+	net_buf_simple_add_u8(&msg, (uint8_t)val);
+
+	return bt_mesh_msg_ackd_send(cli->model, &ctx, &msg, !status ? NULL : &rsp_ctx);
 }
 
 int bt_mesh_brg_cfg_cli_bridging_table_size_get(uint16_t net_idx, uint16_t addr, uint16_t *size)
 {
-	return 0;
+	BT_MESH_MODEL_BUF_DEFINE(msg, OP_BRIDGING_TABLE_SIZE_GET, 0);
+	struct bt_mesh_msg_ctx ctx = BT_MESH_MSG_CTX_INIT_DEV(net_idx, addr);
+	const struct bt_mesh_msg_rsp_ctx rsp_ctx = {
+		.ack = &cli->ack_ctx,
+		.op = OP_BRIDGING_TABLE_SIZE_STATUS,
+		.user_data = size,
+		.timeout = msg_timeout,
+	};
+
+	bt_mesh_model_msg_init(&msg, OP_BRIDGING_TABLE_SIZE_GET);
+
+	return bt_mesh_msg_ackd_send(cli->model, &ctx, &msg, !size ? NULL : &rsp_ctx);
 }
 
 int bt_mesh_brg_cfg_cli_bridging_table_add(uint16_t net_idx, uint16_t addr, uint8_t directions,
```

Comment:
We should use `bridging_table_entry` struct here to reduce count of params. It is generally not efficient to pass more than 4 params to function call unless absolutely needed.

Reply:
I agree.